alerts/absent_alerts.libsonnet

```diff
@@ -0,0 +1,46 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'ceph-absent',
+        rules: [
+          {
+            alert: 'CephMgrIsAbsent',
+            expr: |||
+              absent(up{%(cephExporterSelector)s} == 1)
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'Ceph Manager has disappeared from Prometheus target discovery.',
```

Comment:
I feel a generic message could be something like `Storage metrics collector service not available anymore`.
This kind of message could fit into k8s and well as specific dashboards properly.